rfcs/63-nursery.md

```diff
@@ -0,0 +1,172 @@
+
+### Preparing a crate to be upstreamed
+
+Before a nursery crate can be accepted to Bevy itself, the following conditions must be met:
+
+1. All items in the `upstreaming` milestone must be complete.
+2. A final release must be published, encompassing all PRs merged to date.
+3. The Bevy maintainers must collectively agree to accept the crate upstream in its current form.
+
+This checklist may be helpful when considering issues for the `upstreaming` milestone:
+
+- [] are the examples and documentation helpful and complete?
+- [] do all features work as intended?
+- [] is the code quality acceptable?
+- [] is there any urgent highly-experimental work that should be completed before attempting to integrate more closely with the rest of the engine?
+
+Crates **do not have to be complete** to be merged into Bevy itself: merely polished and clearly useful.
+
+The PR to merge the crate into Bevy itself should be very simple, and contain only the bare minimum change set required to enable interoperability.
+Further integration work should be done in follow-up PRs.
+If you are an interested community member, raise issues and submit fixes on the nursery repository **before** the crate is being upstreamed.
+
+Once the upstreaming PR is merged, the nursery crate's repo will be archived and any still-relevant issues will be migrated to the main Bevy repo.
+
+### Maintaining upstreamed crates
+
+When a nursery crate is upstreamed into the main engine,
+the core maintainer team of the nursery crate will be offered (but do not need to accept)
+**scoped merge rights** for the Bevy project.
+
+Nursery crate maintainers have shown themselves to have the judgement and reliability needed to manage part of the core engine:
+upstreaming a crate should not impose any additional barriers to the crate's development and maintenance.
+Following the standard rules for community reviews, they will be able to merge PRs that affect their nursery crate and directly related areas of the engine.
+Controversial PRs in their area will require either the sign-off of the project lead *or* the nursery crate maintainer, although like always, consulting experts is always wise!
+
+Like always, the project lead may choose to hand out (or revoke) additional permissions on a case-by-case basis where warranted, but the expansion of merge rights outlined above is the default process when a nursery crate is upstreamed.
+
+## Drawbacks
+
+1. Clearly picks winners within the crate ecosystem.
+   1. These winners typically already exist, but it's hard for beginners to discover that critical information.
+2. Hands out additional merge rights to the engine.
+   1. May increase chaos and dilute product direction.
+   2. Tracking areas of expertise may get onerous.
+   3. Marginally increases security risks (2FA + only write permissions) reduce this risk substantially.
+3. Likely to increase the scope of the core engine.
```

Comment:
I'm glad you called it out, but its worth mentioning (at least, via pull request comment) that nursery crates should never expand the "planned" scope of the engine (ex: crates in the nursery would have been manually implemented in-engine eventually). So I don't see this a concerning drawback.